src/core/evaluator.js

```diff
@@ -377,6 +380,34 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
+    const standardFontNameToFileName = getStdFontNameToFileMap();
+    const filename = standardFontNameToFileName[name];
+    if (this.options.standardFontDataUrl !== null) {
+      const url = `${this.options.standardFontDataUrl}${filename}.pfb`;
+      const response = await fetch(url);
+      if (!response.ok) {
+        warn(
+          `fetchStandardFontData failed to fetch file "${url}" with "${response.statusText}".`
+        );
+        return null;
+      }
+      return new Stream(await response.arrayBuffer());
```

Comment:
Sorry, but I still think that you need to properly catch any errors and always fallback to a main-thread fetch here to prevent any future breakage in the GENERIC viewer; something along these lines:
```suggestion
      try {
        const response = await fetch(url);
        if (!response.ok) {
          throw new Error(response.statusText);
        }
        return new Stream(await response.arrayBuffer());
      } catch (e) {
        warn(
          `fetchStandardFontData failed to fetch url "${url}" with "${e}".`
        );
      }
```

Reply:
Did you see my [comment above](https://github.com/mozilla/pdf.js/pull/12726#discussion_r641715489) (may have been lost in all the updates). If you did, and think we should still fallback, we could alternatively only fallback for the generic viewer.

Reply:
Sorry, I completely missed that!

Well, you obviously do have a point about both `fetch`-support and silent failures, and users can of course disable this easily.
Let's just ignore my comment for now, and we can always revisit this (e.g. for `GENERIC`-builds only) if it turns out to cause problems in practice :-)